doc/api/dns.markdown

```diff
@@ -61,14 +61,18 @@ AAAA (IPv6) record. `options` can be an object or integer. If `options` is
 not provided, then IP v4 and v6 addresses are both valid. If `options` is
 an integer, then it must be `4` or `6`.
 
-Alternatively, `options` can be an object containing two properties,
-`family` and `hints`. Both properties are optional. If `family` is provided,
-it must be the integer `4` or `6`. If `family` is not provided then IP v4
-and v6 addresses are accepted. The `hints` field, if present, should be one
-or more of the supported `getaddrinfo` flags. If `hints` is not provided,
-then no flags are passed to `getaddrinfo`. Multiple flags can be passed
-through `hints` by logically `OR`ing their values. An example usage of
-`options` is shown below.
+Alternatively, `options` can be an object containing these properties:
+
+* `family` {Number} - Record family. If present, must be the integer
+  `4` or `6`. If not provided then IP v4 and v6 addresses are accepted.
+* `hints`: {Number} - If present, should be one or more of the supported
+  `getaddrinfo` flags. If `hints` is not provided, then no flags are passed to
+  `getaddrinfo`. Multiple flags can be passed through `hints` by logically
+  `OR`ing their values.
+* `all`: {Boolean} - Instead of returning a single address in the callback,
```

Comment:
This describes how the callback arguments are changed, but doesn't really describe the purpose of `all`. Can you add something like "Return all matching addresses when true"